target/linux/ath79/image/nand.mk

```diff
@@ -231,6 +231,23 @@ define Device/netgear_ath79_nand
   UBINIZE_OPTS := -E 5
 endef
 
+define Device/netgear_pgzng1
+  SOC := ar9344
+  DEVICE_MODEL := PGZNG1
+  DEVICE_VENDOR := NETGEAR
```

Comment:
You could specify an alternate vendor + product through the DEVICE_ALT0_VENDOR + DEVICE_ALT0_MODEL name for the "ADT Pulse Gateway" branding.